src/ui-services/notificationsService.js

```diff
@@ -17,9 +17,9 @@ angular.module('openshiftCommonUI').provider('NotificationsService', function()
       return 'hide/notification/' + namespace + '/' + notificationID;
     };
 
-    var addNotification = function (notification, notificationID, namespace) {
-      if (notificationID && isNotificationPermanentlyHidden(notificationID, namespace)) {
-        notification.hidden = true;
+    var addNotification = function (notification) {
```

Comment:
@jeff-phillips-18 I changed the signature here because I need to be able to get the ID later for `isNotificationVisible`. I don't think we were using this anywhere with a `notificationID` yet, correct?